### PR TITLE
Add airflow support

### PR DESCRIPTION
**Pull request by vidma**
For now we tested only Google's Airflow (Google Cloud Composer).

Use `Hide whitespace (changes)`

docs here https://github.com/kensuio-oss/kensu-py/tree/f/airflow/docs/airflow

### CODE REVIEW

File: kensu/utils/helpers.py [hunk 1]
```diff
@@ -87,6 +87,38 @@ def report_all2all_lineage(in_obj, out_obj, op_type, in_inmem=True, out_inmem=Tr
...
                                                from_col=str(col),
                                                type=op_type)
+
+
+def report_simple_copy_with_guessed_schema(input_uri,  # type: str
+                                           output_absolute_uri,  # type: str
+                                           read_schema_from_filename,  # type: str
+                                           operation_type,  # type: str
+                                           ):
+    from kensu.utils.dsl.extractors.external_lineage_dtos import GenericComputedInMemDs
```
Comment (vidma) [inline kensu/utils/helpers.py:97]: this is same as it was in https://github.com/kensuio-oss/kensu-py/pull/102/files#diff-b1a693175196936a4497753c9d6f4987374273e7c126d77f8c6a3e6172005b17R56-L77 , just extracted

File: kensu/airflow/operators/bash.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            output_absolute_uri = get_absolute_path(output_filename)
+            report_simple_copy_with_guessed_schema(
+                input_uri=input_uri,
+                output_absolute_uri=output_absolute_uri,
+                read_schema_from_filename=output_filename,
+                operation_type='airflow.BashOperator::curl'
+            )
+            return True
+    except Exception as e:
+        logging.warning(f"caught exception in dumb_parse_curl", e)
```
Comment (leszekgruchala) [inline kensu/airflow/operators/bash.py:33]: error?

File: kensu/google/cloud/bigquery/job/query.py [hunk 3]
```diff
@@ -55,73 +55,114 @@ def wrapper(*args, **kwargs):
...
+            register_output_orig_data=is_ddl_write   # used for DDL writes, like CREATE TABLE t2 AS SELECT * FROM t1
+        )
+        if is_ddl_write:
+            if len(lineage.lineage) > 0:
+                kensu.report_with_mapping()
+                return True
+            else:
+                logger.warning("Kensu got empty lineage - not reporting")
+                logger.info("lineage:" + str(lineage))
+                logger.info("ddl_target_table:" + str(ddl_target_table))
```
Comment (leszekgruchala) [inline kensu/google/cloud/bigquery/job/query.py:162]: Shouldn't those be a single warning log?

File: kensu/airflow/operators/bash.py (hunk 2 above)
Comment (vidma) [reply kensu/airflow/operators/bash.py:33]: this one is very experimental/dumb "parser" function, it's not expected to work in general case. so I'd say it's fine

**Review by leszekgruchala — APPROVED** [review]

**Review by leszekgruchala — APPROVED** [review]

**Pull request merged**